### PR TITLE
DTPORTAL-13655 Updates for PHP 7 compatibility

### PR DESCRIPTION
**Pull request by yadnad**
## [JIRA](https://dialogtech.atlassian.net/browse/DTPORTAL-13655)
Update the Rediska session handler to be PHP 7 compatible.

### Requires
- N/A

### Security Checklist
- [x] XSS
- [x] SQL injection

### Code Reviewers
@sfgeorge please review. We created a new `dialogtech/support/0.5.7` branch for the PHP 7 updates in this PR. I couldn't get composer to play nice by changing the version in CRBN and Ifbyphone at the same time. 

We'll need to coordinate with @jalubke when we update the Ifbyphone library for Organic deploys and updating the `composer.lock` file in CRBN.

### CODE REVIEW

**Review by jbuehring — APPROVED** [review]
:crown:

**Comment by sfgeorge** [comment]
This looks great from a content perspective, as this is [content-equivalent to the cloudvox:feature/php7 branch](https://github.com/cloudvox/Rediska/blob/feature/php7/library/Rediska/Zend/Session/SaveHandler/Redis.php#L123-L125).

However, I'd recommend targetting the branch dialogtech/support/0.5.6 rather than a dialogtech/support/0.5.7.  [The content of the dialogtech/support/0.5.7 does not include several of the features that were released with v0.5.7](https://github.com/cloudvox/Rediska/compare/dialogtech/support/0.5.7...shumkov:v0.5.7), so we may cause confusion for ourselves in the future by inferring that we're running a mod'd version of v0.5.7 when we're really running a mod'd version of v0.5.6.

Side note:  Adding confusion to the situation, a previous maintainer of this forked repo began creating their own tagging scheme - using tags like "0.5.8" and "0.5.9", which have no relation whatsoever of the official tags of "v0.5.8" and "v0.5.9".  Watch out when viewing this seemingly duplicative tagging scheme.

**Comment by yadnad** [comment]
@sfgeorge Thanks for the recommendation. In that scenario we'd be treating `dialogtech/support/0.5.6` as our `master` branch and always merging changes into it then?

**Review by sfgeorge — APPROVED** [review]
This looks good by me, thanks man 👍 

@yadnad For now, yeah it would be treating our dialogtech/support/0.5.6 branch as our production branch, a.k.a. master.

However, if at some point in time in which we would be prepared to introduce the changes from v0.5.7 while still keeping our customizations/fixes, then we could create dialogtech/support/0.5.7 based on v0.5.7, and then merge dialogtech/support/0.5.6 into dialogtech/support/0.5.7.

The benefit of doing it this way is that we are able to see in history clearly how close to the original shumkov/Rediska fork we are as we are never more than a handful of commits apart from an officially tagged version - and this makes it easier to reconcile and minimize fork deviation in the future.

**Comment by yadnad** [comment]
Thanks for the great explanation as always @sfgeorge 💯 

@jalubke this is good to merge when you're ready for an Organic deploy

**Pull request merged**
